src/main/java/run/halo/app/model/properties/CommentProperties.java

```diff
@@ -48,7 +48,7 @@ public enum CommentProperties implements PropertyEnum {
      * Dependent comment js.
      */
     INTERNAL_PLUGIN_JS("comment_internal_plugin_js", String.class,
-        "//cdn.jsdelivr.net/npm/halo-comment@latest/dist/halo-comment.min.js"),
+        "//unpkg.com/halo-comment-normal@1.5.0-alpha.0/dist/halo-comment.min.js"),
```

Comment:
```suggestion
        "https://unpkg.com/halo-comment@latest/dist/halo-comment.min.js"),
```

默认还是用这个吧，halo-comment-normal 这个库是另外一个选择。